bittensor/_cli/commands/overview.py

```diff
@@ -338,5 +338,8 @@ def check_config( config: 'bittensor.Config' ):
             config.wallet.name = str(wallet_name)
 
         if config.netuid != []:
-            config.netuid = [int(netuid) for netuid in config.netuid]
+            if not isinstance(config.netuid, list):
```

Comment:
Is this change only for the overview or for all of the cli commands?

Reply:
This is only present for overview to allow restricting to specific netuids, instead of querying the metagraph for all. 
The issue was the API used an int for `config.netuid` and I had changed the arg to a `List[int]`, so this check fixes the broken expectation.